ephios/core/services/notifications/types.py

```diff
@@ -452,11 +452,16 @@ class CustomEventParticipantNotification(AbstractNotificationHandler):
     @classmethod
     def send(cls, event: Event, content: str):
         participants = set()
+        responsible_users = get_users_with_perms(
+            event, with_superusers=False, only_with_perms_in=["change_event"]
+        )
         for shift in event.shifts.all():
             participants.update(shift.get_participants())
         notifications = []
         for participant in participants:
```

Comment:
instead of using the if statement below, you could also use the [queryset difference](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#difference) to make it a bit more readable/understandable what happens here. But it's also fine the way it is

Reply:
How would one do that? How would the difference look here? I think that would be far more complex...

Reply:
`participants.difference(responsible_users)`?

Reply:
I think you are mistaken, participants is not a queryset. Participants aren't even model instances.

Reply:
ah yeah